crates/biome_js_type_info/tests/utils/mod.rs

```diff
@@ -40,23 +71,7 @@ impl TypeResolver for PromiseResolver {
     }
 }
 
-pub(crate) fn get_expression_statement(root: &AnyJsRoot) -> JsExpressionStatement {
```

Comment:
I moved this function inside `flattening` because of some reason clippy was marking it as unused